tests/test_generics.py

```diff
@@ -131,7 +143,7 @@ class Config:
 def test_generic_instantiation_error():
     with pytest.raises(TypeError) as exc_info:
         GenericModel()
-    assert str(exc_info.value) == 'Type GenericModel cannot be instantiated; it can be used only as a base class'
+    assert str(exc_info.value) == 'Type GenericModel cannot be used without generic parameters, e.g. GenericModel[T]'
```

Comment:
This might be slightly confusing because you also have to inherit from Generic before this will work, but if you try (e.g. by doing GenericModel[int]) it will then tell you to inherit from typing.Generic. So I think it's okay.